nixos/modules/services/web-servers/garage.nix

```diff
@@ -53,24 +59,21 @@ in
             description = "The main data storage, put this on your large storage (e.g. high capacity HDD)";
           };
 
-          replication_mode = mkOption {
-            default = "none";
-            type = types.enum ([ "none" "1" "2" "3" "2-dangerous" "3-dangerous" "3-degraded" 1 2 3 ]);
-            apply = v: toString v;
-            description = "Garage replication mode, defaults to none, see: <https://garagehq.deuxfleurs.fr/documentation/reference-manual/configuration/#replication-mode> for reference.";
```

Comment:
Removed options should get a corresponding removal warning using `lib.mkRemovedOptionModule`.